applications/acdc/test/test.config

```diff
@@ -0,0 +1,10 @@
+[
+ {lager, [
+          {handlers, [
+                      {lager_console_backend, '=info'}
```

Comment:
@fenollp to be clear, what exactly did you run into? Something with the line in my above test.config? Or in your test-config branch did you happen to have a problem?